cmd/metricsbridge/example-config.yaml

```diff
@@ -1,4 +1,4 @@
-intervalNanoseconds: 30000000000 # every 30 seconds
```

Comment:
not sure how this would work because of prometheus scrape intervals. Prometheus scrapes each 30s, and we do each 10s. Need more reading on how it works :/